package.json

```diff
@@ -20,37 +20,36 @@
         "registry": "https://registry.npmjs.org"
```

Comment:
Please update version to 7.3.0 in package.json and run npm install

Reply:
Updated version and re-checked all dependencies